tests/foreman/longrun/test_oscap.py

```diff
@@ -598,11 +700,15 @@ def test_positive_oscap_run_via_local_files(
 
     :CaseImportance: Critical
     """
+    contenthost = rex_contenthost
+    os_version = contenthost.os_version.major
+    distro = f'rhel{os_version}'
     SELECTED_ROLE = 'theforeman.foreman_scap_client'
     file_name = 'security-data-oval-com.redhat.rhsa-RHEL8.xml.bz2'
     download_url = 'https://www.redhat.com/security/data/oval/v2/RHEL8/rhel-8.oval.xml.bz2'
     profile = OSCAP_PROFILE['ospp8']
```

Comment:
I think this should be:
```suggestion
    profile = profiles[distro]
```

Which you already do on line 736